README.md

```diff
@@ -431,6 +432,10 @@ A curated list of awesome Java frameworks, libraries and software.
 * [IntelliJ IDEA ![c]](http://www.jetbrains.com/idea/) - Supports a lot of JVM languages and provides good options for Android development. The commercial edition targets the enterprise sector.
 * [NetBeans](https://netbeans.org/) - Provides integration for several Java SE and EE features from database access to HTML5.
 
+## IntelliJ IDEA Java Plugins
```

Comment:
Link in content.
Description.
2 more plugins - not just any plugins. Viable and used ones please.